### DISCUSSION TITLE
upscayl CLI gives worse quality than GUI version

### DISCUSSION THREAD

**Discussion by jahurulislam741** [upscayl/upscayl#720] [Q&A]
Hi there, 

I am using CLI version upscayle-ncnn (https://github.com/upscayl/upscayl-ncnn), as of Feb. 16, 2024, but the output quality appears to be considerably poorer than the GUI version (v2.9.9). I was curious how it could be improved in the CLI version. 

Thank you.

**Comment by aaronliu0130** [comment]
Could you provide an example? Did you use the same model?
